components/navigable-container/README.md

```diff
@@ -0,0 +1,156 @@
+NavigableContainer
+=============
+
+`NavigableContainer` is a React component to render a container navigable using the keyboard. Only things that are focusable can be navigated to. It will currently always be a `div`.
+
+`NavigableContainer` is exported as three convenience classes: `NavigableMenu`, `NavigableGrid` and `TabbableContainer`. `NavigableContainer` itself is not exported. `NavigableMenu`, `NavigableGrid`, and `TabbableContainer` have some shared props, and some individual props. Any other props will be passed through to the `div`.
+
+----
+
+## Shared Props
+
+These are the props that `NavigableMenu`, `NavigableGrid` and `TabbableContainer` all share.
+
+
+### initialSelector
+
+A selector which tells the component what to focus first inside it. If the selector is not matched, the first focusable item will be focused. This can be useful to focus the previously selected descendant when this component get focused.
+
+- Type: `String`
+- Required: No
+
+### onNavigate
+
+A callback invoked when the menu navigates to one of its children passing the index and child as an argument
+
+- Type: `Function`
+- Required: No
+
+### deep
+
+A boolean to look for navigable children in the direct children or any descendant. True means that any descendant can be considered navigable, and false means only direct children are considered.
+
+- Type: `Boolean`
+- Required: No
+- default: false
+
+
+### widget
+
+A boolean which specifies whether or not the focusable elements in this component might contain the focus, rather than have the focus. This is often used for `TabbableContainer` layouts that might have arrow key navigation inside each tabstop.
+
+- Type: `Boolean`
+- Required: No
+- default: true for `TabbableContainer`, false otherwise.
+
+
+## NavigableMenu
+
+
+A NavigableMenu allows movement up and down (or left and right) the component via the arrow keys. The `tab` key is not handled. The `orientation` prop is used to determine whether the arrow keys used are vertical or horizontal.
+
+### Usage
+
+```jsx
+import { NavigableMenu, Button } from '@wordpress/components';
+
+function MyMenu() {
+	return (
+		<NavigableMenu>
+			<Button>My Button 1</Button>
+			<Button>My Button 2</Button>
+			<Button>My Button 3</Button>
+		</NavigableMenu>
+	);
+}
+```
+
+### Additional Props
+
+#### orientation
+
+The orientation of the menu. It could be "vertical" or "horizontal"
+
+- Type: `String`
+- Required: No
+- Default: `"vertical"`
+
+#### stopArrowKeys
```

Comment:
Can we find a better name? `preventBubblingArrowEvents` or something. I'm bad at naming things though :)

Reply:
That name sounds slightly better :)